exercises/phone-number/lib/phone_number.dart

```diff
@@ -0,0 +1,3 @@
+class PhoneNumber {
+  // Put your code here.
```

Comment:
Given the introduction to dynamic is in this. I wonder how you might feel about add a "tip" in the README or going ahead and defining the function in the class (but don't return anything. Just have a brief description of dynamic with a link to a dartlang.org page about it.

Reply:
That would be nice, I'll add a hint in the Readme only and keep the class empty.